README.md

```diff
@@ -1,34 +1,40 @@
 [![PyPI version](https://badge.fury.io/py/django-admin-autocomplete-filter.svg?kill_cache=1)](https://badge.fury.io/py/django-admin-autocomplete-filter)
 
+
 Django Admin Autocomplete Filter
 ================================
-A simple Django app to render list filters in django admin using autocomplete widget. This app is heavily inspired from [dal-admin-filters.](https://github.com/shamanu4/dal_admin_filters)
+A simple Django app to render list filters in django admin using an autocomplete widget. This app is heavily inspired by [dal-admin-filters.](https://github.com/shamanu4/dal_admin_filters)
+
 
 Overview:
 ---------
 
 Django comes preshipped with an admin panel which is a great utility to create quick CRUD's.
-The django 2.0 came with much needed [autocomplete_fields](https://docs.djangoproject.com/en/2.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.autocomplete_fields "autocomplete_fields") which uses select2 widget that comes with a search feature that loads the options asynchronously.
-We can use this in django admin list filter.
+Version 2.0 came with a much needed [`autocomplete_fields`](https://docs.djangoproject.com/en/2.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.autocomplete_fields "autocomplete_fields") property which uses a select2 widget to load the options asynchronously.  We leverage this in `django-admin-list-filter`.
 
     
 
 Requirements:
 -------------
+
 Requires Django version >= 2.0
 
+
 Installation:
 -------------
-You can install it via pip or to get the latest version clone this repo.
+
+You can install it via pip.  To get the latest version clone this repo.
```

Comment:
I would say that the repo contains the latest changes, but by definition the latest version is the latest one on PyPI.

But installing from the repo is not the best practice for Python projects, and should generally only be needed to help with development, or when a project is slow to release changes to PyPI.